client/public/locales/en/translation.json

```diff
@@ -526,7 +526,7 @@
   },
   "tooltip": {
     "priority": "Tasks priority, a non-negative number, impacts tasks scheduling policy. Lowest priority is 0. Higher priority tasks run before lower priority tasks.",
-    "preemption": "If enabled, allows the scheduler to cancel a running task and free the resources for higher priority tasks."
+    "preemption": "If enabled, allows the scheduler to re-schedule a running task and free the resources for higher priority tasks."
```

Comment:
recommend: 
"preemption": "If enabled, allows the scheduler to reschedule other running tasks to free cluster resources needed to run higher priority tasks."

Reply:
@jortel 
> reschedule other running tasks
>
This part is not clear for me. Does it mean that preemption flag set on task A affects scheduling of all other tasks (but not directly task A) ?

Reply:
Yes. exactly.